### ISSUE TITLE
ML Models: first categorical classifier (before/after custom dataloader is created)

### ISSUE THREAD

**Issue by mlederbauer** [mlederbauer/NMRcraft#4]
Machine Learning Models
Why do we need it?: to model an implicit relationship of signal + a bit of information of ligands -> "complex with all ligands" (this is a bit convoluted right now, we should start with a simple toy example of predicting the 6th ligand when 5 ligands are available and build from that... we should spend a bit of time in clearly defining the "problem" / "classification task" here.
When do we need it?: also fairly soon to be able to refine
Who should do it?: a simple training script is already written in ./scripts/training/train_simple_RF.py. More complex models can be declared in ./nmrcraft/models/. I think this is a bit more involved in terms of "freedom of what one can do" and requires some coding. But it would be great to start with a really "simple" (= not so many lines of code) random forest classifier. I think ~2 people should be on it
-- Declare model -> Random Forest Classifier? Logistic Multivariate Reg? Decision Tree? Ensemble Model? Completely up to us!
-- Optimize model and hyperparams to achieve a high classification accuracy/ROC
-- Explainability -> feature importance? That would be a super cool thing to implement if we have time
